css/types/counters.json

```
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "attr": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counters",
+          "description": "<code>counters()</code>",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
```

Since Edge is a continuation off of IE, all features in IE carry over to Edge.  Let's set this to the first Edge version:
```suggestion
              "version_added": "12"
```